qa/SidechainTestFramework/account/smart_contract_resources/contracts/WithdrawalRequests.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
```

Comment:
The solidity interface seems not compatible with the native smart contract in the SDK. In this case the file `qa/truffle/contracts/WithdrawalRequests.sol` seems to be the correct one.